### PR TITLE
make buttons edgy on mobile

### PR DESCRIPTION
**Pull request by szaimen**
This is best reviewed like this: https://github.com/nextcloud/firstrunwizard/pull/698/files?diff=unified&w=1

Reason for this: I think edgy buttons look better for mobile:

| Before | After |
|---|---|
| image | image |
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>For my own testing</summary>

```
docker run -it \
-e FIRSTRUNWIZARD_BRANCH=enh/noid/make-buttons-edgy-on-mobile \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.146.128 \
--name nextcloud-easy-test \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -236,10 +236,17 @@
...
-		.modal-mask .modal-wrapper .next, .prev {
-			top: 220px;
-			align-items: start !important;
+		.modal-mask .modal-wrapper {
+			.next, .prev {
+				top: 220px;
+				align-items: start;
+			}
+
+			.icon-next, .icon-previous {
```
Comment (artonge) [inline src/App.vue:247]: Add a comment here too ?

**Review by artonge — APPROVED** [review]
Code looks good. But let's wait for the design team input before merging.

File: src/App.vue (hunk 1 above)
Comment (szaimen) [reply src/App.vue:248]: done

**Review by jancborchardt — CHANGES_REQUESTED** [review]
> Reason for this: I think edgy buttons look better for mobile

:D love the reasoning.

Counter-proposal: We keep the circular buttons, but on mobile we move them to the bottom, with a bit of whitespace to the bottom and each of the sides respectively. Kind of like an Android FAB, but on both sides. Also easier to reach for fingers.

**Comment by szaimen** [comment]
> Counter-proposal: We keep the circular buttons, but on mobile we move them to the bottom, with a bit of whitespace to the bottom and each of the sides respectively. Kind of like an Android FAB, but on both sides. Also easier to reach for fingers.

Prpblem with moving it to the bottom is that the buttons will overlap the content and other buttons in the templates which looks really weird, in my opinion. That is why they are positioned where they are currently...

**Comment by nimishavijay** [comment]
Looks interesting! 
Most of our buttons are round (or have a pill-shape) so I'm wondering if it would be consistent with our designs 🤔

**Comment by szaimen** [comment]
> Looks interesting! Most of our buttons are round (or have a pill-shape) so I'm wondering if it would be consistent with our designs 🤔

Not sure. Please decide and let me know to merge or to close :)

**Comment by szaimen** [comment]
I am fine with any decision but I need a decision here :)

**Comment by nimishavijay** [comment]
I would keep the circular buttons, it is consistent with our style :)

**Comment by szaimen** [comment]
> I would keep the circular buttons, it is consistent with our style :)

okay 👍

**Comment by jancborchardt** [comment]
Sorry for late reply @szaimen  · regardong the overlap issue, we can make sure that if left/right controls are present, that on mobile there's room for them below the modal?

**Pull request closed without merging**